### PR TITLE
Adding Okta Sign-In Widget

### PR DESCRIPTION
**Pull request by joelfranusic-okta**
@eklein-okta: For your review.

### CODE REVIEW

File: _source/docs/guides/okta_sign-in_widget.md [hunk 1]
```diff
@@ -0,0 +1,531 @@
...
+            if (res.status === 'SUCCESS') { res.session.setCookieAndRedirect('https://example.com/'); }
+          }
+        );
+      </script>
+    </body>
+
+Here is what is happening in the HTML above: 
+
+First, in the `<head>`
+tag, we include the  `okta-sign-in-1.0.0.min.js` and
```
Comment (karlmcguinness-okta) [inline _source/docs/guides/okta_sign-in_widget.md]: We should probably find a way to parameterize the version number.  The most current version in 1.0.1 now and we will keep shipping new versions and will not be updating this doc.  Maybe a version history at the top and some standard variable syntax such as `okta-sign-in-{{version}}.min.js`.  This comment applies to pretty every spot in the doc that says the version number.  Thoughts?

File: _source/docs/guides/okta_sign-in_widget.md [hunk 2]
```diff
@@ -0,0 +1,531 @@
...
+      <title>Example Okta SignIn Widget</title>
+    
+      <!--[if lt IE 9]>
+          <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.2/html5shiv.min.js"></script>
+      <![endif]-->
+    
+      <script src="https://example.okta.com/js/mvc/login/okta-login.js" type="text/javascript"></script>
+      <link href="https://example.okta.com/css/login/okta-login.css" type="text/css" rel="stylesheet">
+      <link href="https://example.okta.com/css/login/okta-theme.css" type="text/css" rel="stylesheet">
+      <link href="https://fonts.googleapis.com/css?family=Open+Sans:400" type="text/css" rel="stylesheet">
```
Comment (karlmcguinness-okta) [inline _source/docs/guides/okta_sign-in_widget.md]: We can probably kill the custom font in this example

File: _source/docs/guides/okta_sign-in_widget.md [hunk 3]
```diff
@@ -0,0 +1,531 @@
...
+started on TCP port 8000 of your local machine).
+
+## Testing the Okta Sign-In Widget
+
+At this point, you are ready to test the Okta Sign-In Widget. 
+
+Test the Okta Sign-In Widget by loading the URL for the
+`login-to-okta.html` file in your favorite web browser.
+
+If you used `python -m SimpleHTTPServer` command above, this URL will be
```
Comment (karlmcguinness-okta) [inline _source/docs/guides/okta_sign-in_widget.md]: nit:  this is repeated a few times, consider link to `## Copy the HTML to a web server`

File: _source/docs/guides/okta_sign-in_widget.md [hunk 4]
```diff
@@ -0,0 +1,531 @@
...
+## Configuring CORS support on your Okta organization
+
+The last step that you'll need to do before you can test out
+the Okta Sign-In Widget is to enable CORS for the web server you just set
+up.
+
+You can enable CORS by following the steps in our guide for
+[guide for Enabling CORS](http://developer.okta.com/docs/api/getting_started/enabling_cors.html). Configure CORS using the same URL for the
+web server you are using to host the HTML for the Okta Sign-In Widget. For
+example, if you used the `python -m SimpleHTTPServer` example above, the URL
```
Comment (karlmcguinness-okta) [inline _source/docs/guides/okta_sign-in_widget.md]: nit:  this is repeated a few times, consider link to `## Copy the HTML to a web server`

File: _source/docs/guides/okta_sign-in_widget.md [hunk 5]
```diff
@@ -0,0 +1,531 @@
...
+`login-to-okta.html` file in your favorite web browser.
+
+If you used `python -m SimpleHTTPServer` command above, this URL will be
+<http://localhost:8000/login-to-okta.html>
+
+Once you've successfully loaded the the Okta Sign-In Widget, it is
+time to start exploring the capabilities of the
+widget. 
+
+Here are two things for you to try:
```
Comment (karlmcguinness-okta) [inline _source/docs/guides/okta_sign-in_widget.md]: We should probably have add a blockquote with a link to https://support.okta.com/help/articles/Knowledge_Article/27315047-Configuring-Multifactor-Authentication to point user to product docs for setting up MFA

File: _source/docs/guides/okta_sign-in_widget.md [hunk 6]
```diff
@@ -0,0 +1,531 @@
...
+    </head>
+    <body>
+      <!-- Render the login widget here -->
+      <div id="okta-login-container"></div>
+    
+      <!-- Script to init the widget -->
+      <script>
+        var oktaSignIn = new OktaSignIn({baseUrl: 'https://example.okta.com'});
+    
+        oktaSignIn.renderEl(
```
Comment (karlmcguinness-okta) [inline _source/docs/guides/okta_sign-in_widget.md]: Can we get brief section that docs the renderEl method and callback signatures (e.g. the `res` object that has user and and status properties)?

File: _source/docs/guides/okta_sign-in_widget.md (hunk 5 above)
Comment (joelfranusic-okta) [inline _source/docs/guides/okta_sign-in_widget.md]: I wasn't exactly sure what you were looking for here, so I added a step to this section with a link to the MFA article above.

**Pull request merged**